### PR TITLE
fix: Update pom.xml versions by removing hardcoded values in config.json file

### PR DESCRIPTION
**Pull request by thogarty**
* remove hardcoded values from spec config.json
* run `make generate`

### CODE REVIEW

File: equinix-openapi-fabric/pom.xml [hunk 1]
```diff
@@ -5,9 +5,9 @@
     <artifactId>equinix-openapi-fabric</artifactId>
     <packaging>jar</packaging>
     <name>equinix-openapi-fabric</name>
-    <version>0.5.0</version>
+    <version>0.10.0</version>
     <url>https://github.com/openapitools/openapi-generator</url>
-    <description>Equinix Fabric API Specification 4.11 generated SDK</description>
+    <description>OpenAPI Java</description>
```
Comment (d-bhola) [inline equinix-openapi-fabric/pom.xml:10]: Do we need to add 'SDK' in this description?

Comment (thogarty) [reply equinix-openapi-fabric/pom.xml:10]: That's just default from the open api generator. For now we can leave as is and return to it later with a longer term fix. Just need to get this release cycle handled properly.

Comment (d-bhola) [reply equinix-openapi-fabric/pom.xml:10]: Sure

**Review by d-bhola — APPROVED** [review]

**Review by srushti-patl — APPROVED** [review]

**Comment by tutkat** [comment]
@thogarty you forgot to bump up version in tests pom dependency

**Pull request merged**